docs/build/circuit-construction.ipynb

```diff
@@ -50,6 +50,74 @@
     "qc.qubits"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "392e3d6c",
+   "metadata": {},
+   "source": [
+    "Multiple QuantumRegisters and ClassicalRegisters can be combined to create a circuit. Every [`QuantumRegister`](/api/qiskit/circuit#qiskit.circuit.QuantumRegister) and [`ClassicalRegister`](/api/qiskit/circuit#qiskit.circuit.ClassicalRegister) can also be named."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6160e3f7",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[Qubit(QuantumRegister(2, 'qreg1'), 0),\n",
+       " Qubit(QuantumRegister(2, 'qreg1'), 1),\n",
+       " Qubit(QuantumRegister(1, 'qreg2'), 0)]"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit import QuantumRegister, ClassicalRegister\n",
+    "\n",
+    "qr1 = QuantumRegister(2, 'qreg1') # Create a QuantumRegister with 2 qubits \n",
+    "qr2 = QuantumRegister(1, 'qreg2') # Create a QuantumRegister with 1 qubit\n",
+    "cr1 = ClassicalRegister(3, 'creg1') # Create a ClassicalRegister with 3 cbits \n",
+    "\n",
+    "combined_circ = QuantumCircuit(qr1, qr2, cr1) # Create a quantum circuit with 2 QuantumRegisters and 1 ClassicalRegister\n",
+    "combined_circ.qubits"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e7b212f",
+   "metadata": {},
+   "source": [
+    "The index and register of a qubit can be obtained using the circuit's [`find_bit`](/api/qiskit/qiskit.circuit.QuantumCircuit#qiskit.circuit.QuantumCircuit.find_bit) method and its attributes. "
```

Comment:
I believe in the original issue attached to this PR it was mentioned that a user was especially interested in the `qc.qubits.index(your_qubit)` method of getting information. could you include something about this as well as the `find_bit` method?

Reply:
I didn't include that due to [this comment in a previous PR ](https://github.com/Qiskit/documentation/pull/1587#discussion_r1649463291) which says it's not recommended due to the significant performance overhead. Please let me know whether to include it or not in reference to this, thanks!

Reply:
I agree with @shraddha-aangiras. The two approaches have the same output, but `find_bit` is more efficient.